### DISCUSSION TITLE
Is there a more widely maintained tagging program with similar functionality that I'm not aware of?

### DISCUSSION THREAD

**Discussion by benpage2** [beetbox/beets#5176] [General]
Beets seems to have a a lot less activity given its feature set that I think it should. Is there another program that's become more popular?

**Comment by benpage2** [comment]
Guess I'm sticking around. Can't find anything else as good.

**Reply by Serene-Arc** [reply]
Are you looking at the development branch? We still have a very active codebase and userbase, we just haven't had a release in a couple years (something I'm working on). But beets is still very much alive.

**Reply by benpage2** [reply]
I don't know. I'll try and find it on my own but if you wouldn't mind pointing me in that direction that would be super helpful.

**Reply by Serene-Arc** [reply]
If you mean for on here, you can see the newest code on this site. It's the development branch on GitHub. When you're on the project page, there's a little box at the top that should normally say 'master'; there's 'development' when you click on it.

For the docs, there should be a box in the very lower left that normally says 'stable'. Switch that to 'latest' and you'll see the docs for the latest version; it's a big change and the theme changes as well, so you can't miss when you have it.

For installing beets to actually use, you can pull directly from the development branch with `pip install -U git+https://github.com/beetbox/beets.git#development` and it'll get the latest version for you and install it. You can just run that again whenever you want to get the latest features, at least until we get the releases going again.

**Reply by benpage2** [reply]
So maybe I'm missing something obvious but I can't find it. I can see 28 total branches, but none are active other than the master and the development branch isn't showing up. Although, I was able to find the docs!

**Reply by Serene-Arc** [reply]
My mistake, I got confused with another project. There is no development branch, we merge directly into master. So master is the best record of the current features and you can install it directly with `pip install -U git+https://github.com/beetbox/beets.git`

**Reply by benpage2** [reply]
What's really funny about this entire convo is I'm actually running the master branch already.

Although, I had zero idea about the development docs. Thanks so much for pointing it out :)

Also a great reminder that I probably need to update it

**Reply by Serene-Arc** [reply]
Great :) Hopefully you like the new features!
